docs/detections/api/rules/rules-api-overview.asciidoc

```diff
@@ -31,6 +31,18 @@ the status of Elastic <<prebuilt-rules, prebuilt rules>>
 TIP: You can view and download a Detections API Postman collection
 https://github.com/elastic/examples/tree/master/Security%20Analytics/SIEM-examples/Detections-API[here].
 
+[float]
+=== Authentication
+This API supports both key- and token-based authentication.
+
+To use key-based authentication, create an {kibana-ref}/api-keys.html[API key], then specify the key in the header of your API calls.
+
+To use token-based authentication, provide a username and password; this automatically creates an API key that matches the current user's privileges.
+
+In both cases, the API key is subsequently used for authorization when the rule runs.
+
+NOTE: If the API key has different privileges than the key that created or most recently updated the rule, the rule behavior might change.
```

Comment:
I noticed that here we talk about rule behavior change and in the other warnings we talk about API key deletion and the resulting consequence. Do we want to have a single warning/note to address both scenarios - 1) key is deleted and rule begins failing 2) user has different privileges than may be expected and rule behavior differs?

Reply:
Makes sense, @yctercero! I've updated the notes to cover both.